src/bittrex-client.js

```diff
@@ -110,14 +110,16 @@ class BittrexClient {
    * @param {String|Number} options.price
    * @return {Promise}
    */
-  async buyLimit(market, { quantity, rate } = {}) {
+  async buyLimit(market, { quantity, rate, timeInForce } = {}) {
```

Comment:
To prevent from from a breaking change please set the default:

```suggestion
  async buyLimit(market, { quantity, rate, timeInForce = 'GTC' } = {}) {
```

Reply:
didn't notice that, will make the changes